cli/flags.rs

```diff
@@ -53,6 +103,7 @@ To run the REPL:
 
 To execute a sandboxed script:
 
+  deno https://deno.land/welcome.ts
   deno run https://deno.land/welcome.ts
```

Comment:
I think it's better to just give one command here - this is the intro - so let's not confuse people with all the possible combinations. I suggest we use the "non-run" version for this one.

Consider updating the example on website/index.html too